csharp/src/Microsoft.ML.OnnxRuntime/NativeMethods.cs

```diff
@@ -160,9 +160,8 @@ public struct OrtApi
         public IntPtr GetStringTensorElementLength;
         public IntPtr GetStringTensorElement;
         public IntPtr FillStringTensorElement;
-        public IntPtr EnablePrePacking;
-        public IntPtr DisablePrePacking;
-        
+        public IntPtr AddSessionConfigEntry;
+
```

Comment:
Wouldn't have expected this change to be valid based on the comment on lines 15 and 16. I thought we had to be careful about ordering here but perhaps that has changed.

Reply:
The order after the change is the same as the onnxruntime_c_api.cc, which is valid?

---
In reply to: [469198522](https://github.com/microsoft/onnxruntime/pull/4718#discussion_r469198522) [](ancestors = 469198522)